### PR TITLE
Recording helper methods

### PR DESCRIPTION
**Pull request by Tzyinc**
created now struct TimeTracker
 on init, it saves the current date time (according to swift documentation sub-millisecond accuracy)

on setTimePadPair, it takes the current time and pairs it with an array of pads. the function will truncate any time that's outside the tolerance level (hardcoded to 1/60 of a second)

### CODE REVIEW

**Comment by anand-sundaram** [comment]
The recording is tied to a session, right? If the session has multiple pages then the indexPath is not enough to uniquely identify the pad..

File: MIDIchlorians/MIDIchlorians/TimeTracker.swift [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+//  MIDIchlorians
+//
+//  Created by Ten Zhi yang on 1/4/17.
+//  Copyright © 2017 nus.cs3217.a0118897. All rights reserved.
+//
+
+import Foundation
+
+struct TimeTracker {
+    private var timeIndexDict: [TimeInterval: [IndexPath]]
```
Comment (Tzyinc) [inline MIDIchlorians/MIDIchlorians/TimeTracker.swift]: page and index path

**Comment by advaypal** [comment]
@Tzyinc Is this good to go?

**Comment by Tzyinc** [comment]
no, will submit a new PR, this is outdated and conflicts with the pbxproj that I dont want to touch

**Comment by advaypal** [comment]
@Tzyinc I think you can close this then

**Pull request closed without merging**